src/scales.js

```diff
@@ -140,8 +141,16 @@ function inferScaleType(key, channels, {type, domain, range}) {
 }
 
 // Positional scales default to a point scale instead of an ordinal scale.
-function asOrdinalType(key) {
-  return registry.get(key) === position ? "point" : "ordinal";
+function asOrdinalType(key, type = "categorical") {
```

Comment:
It’s a little surprising to see categorical here as the default, as it only applies to color scales.

Reply:
True. And in any case my analysis was flawed, since in this part type is always undefined!